lbrynet/dht/node.py

```diff
@@ -425,14 +421,22 @@ def parseResults(nodes):
         return df
 
     @rpcmethod
-    def ping(self):
+    def ping(self, **kwargs):
         """ Used to verify contact between two Kademlia nodes
 
         @rtype: str
         """
         return 'pong'
 
     @rpcmethod
+    def echo(self, value, **kwargs):
```

Comment:
why do we need this?

Reply:
I'd initially fixed it because it was necessary to get testRPCRequestArgs to pass. Since it's not one of the 4 rpc commands in the kademlia spec I removed it and updated the test to use `ping`.